corner_detector:
  # map_file directory
  # map_file: 

  # Use contour approximation if true
  use_approximation: True

  # Contour approximation parameter
  approx_param: 0.005

  # Distance for contour point removal
  distance_threshold: 5

  # Determines the index of convex hull sorted by its area
  convex_hull_idx: 2